frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx

```diff
@@ -15,25 +17,34 @@ function ChangePercentage({
 	direction,
 	duration,
 }: ChangePercentageProps): JSX.Element {
-	if (!percentage || !duration) {
-		return <div className="change-percentage" />;
-	}
-	return (
-		<div
-			className={`change-percentage ${
-				direction > 0 ? 'change-percentage--success' : 'change-percentage--error'
-			}`}
-		>
-			<div className="change-percentage__icon">
-				{direction > 0 ? (
+	if (direction > 0) {
+		return (
+			<div className="change-percentage change-percentage--success">
+				<div className="change-percentage__icon">
 					<ArrowDownLeft size={14} color="var(--bg-forest-500)" />
-				) : (
-					<ArrowUpRight size={14} color="var(--bg-cherry-500)" />
-				)}
+				</div>
+				<div className="change-percentage__label">
+					{percentage}% vs Last {duration}
+				</div>
 			</div>
-			<div className="change-percentage__label">
-				{percentage}% vs Last {duration}
+		);
+	}
+	if (direction < 0) {
+		return (
+			<div className="change-percentage change-percentage--error">
+				<div className="change-percentage__icon">
+					<ArrowUpRight size={14} color="var(--bg-cherry-500)" />
+				</div>
+				<div className="change-percentage__label">
+					{percentage}% vs Last {duration}
```

Comment:
use % icon from lucide

Reply:
in the designs, we have text % (https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7734&m=dev)